srcpkgs/libcamera/template

```diff
@@ -0,0 +1,12 @@
+# Template file for 'libcamera'
+pkgname=libcamera
+version=0.0.3
+revision=1
+build_style=meson
+hostmakedepends="meson ninja pkg-config libyaml-devel python3-yaml python3-ply python3-Jinja2 openssl openssl-devel libevent-devel gst-plugins-base1-devel gstreamer1-devel"
```

Comment:
```suggestion
hostmakedepends="meson ninja pkg-config libyaml-devel python3-yaml python3-ply python3-Jinja2"
makedepends="openssl-devel libevent-devel gst-plugins-base1-devel gstreamer1-devel"
```

Reply:
This commit readded meson and ninja to hostmakedepends, so I made another where I moved openssl etc. to makedepends without readding meson and ninja. I hope that's ok?